src/components/displaybooks.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable */
```

Comment:
- [ ] Kindly remove the `eslint` disabling comment at the beginning of this file. It is not good JavaScript practice to use eslint disabling comments. `Eslint` helps developers write clean and industry-standard JavaScript code. 👍

Reply:
- Hey, please, as suggested by the previous reviewer, kindly remove the `eslint` disable comment here as he's stated the reason above. Please, kindly work on this issue :pray: